Makefile.pre.in

```diff
@@ -1309,6 +1309,11 @@ check-abidump: all
 
 .PHONY: regen-limited-abi
 regen-limited-abi: all
+	# Regenerate files using using Tools/build/stable_abi.py:
+	# - Doc/data/stable_abi.dat
+	# - Lib/test/test_stable_abi_ctypes.py
+	# - Modules/_testcapi_feature_macros.inc
+	# - PC/python3dll.c
```

Comment:
Do you volunteer to keep this list up to date?

Reply:
It's ok if it's outdated, it's just a comment.

Reply:
If it's not necessary, please remove it. You can say how to get the up-to date info, but I don't think that's needed either.

Reply:
I added the URL that you proposed in PR #105351.